content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment.
+ 
+
+\begin{itemize}
+
+\item
+In a thread-compliant implementation, an \openshmem{} \ac{PE} is multithreaded. 
+Each thread can issue \openshmem{} calls; however the threads
```

Comment:
@nspark What do you think about this to address your concern ? "A multithreaded OpenSHMEM PE using thread-compliant implementation can issue OpenSHMEM calls from any of its threads. However, the threads are not separately addressable."